Trust and compliance: Security section (#104)
pretix/pretix-docs: docs/trust/security/pci-dss.md

# PCI DSS

The **Payment Card Industry Data Security Standard** (PCI DSS) mandates security requirements that every party involved with accepting credit cards needs to follow.
Following PCI DSS standards is not required by law, but your payment service provider might require you to send them documentation that you do so.

> Depending on what role you play in the credit card acceptance process, only a subset of the requirements need to be followed.

Reviewer: ```suggestion
Depending on what role you play in the credit card acceptance process, you may only need to follow a subset of the requirements. 
```

> If in doubt, you should ask your payment service provider on what exactly you need to do.

Reviewer: ```suggestion
If in doubt, ask your payment service provider what exactly you need to do.
```

> pretix never handles credit card direction directly.

Reviewer: ```suggestion
pretix never handles credit cards directly.
```
Ich hab den Begriff "credit card direction" noch nie gehört und finde auch nichts dazu

Author: Das muss ein Tippfehler sein. Ich füge was präziseres ein
```suggestion
pretix never handles credit card account data directly.
```
credit cards handeln wir ja schon, "handlen" ist ein sehr weiter begriff

All of our official integrations with payment providers are built in a way that embeds the credit card form from the payment provider either through a redirect or iframe.
Therefore, you usually need to fill out the questionnaire [SAQ A](https://docs-prv.pcisecuritystandards.org/SAQ%20(Assessment)/SAQ/PCI-DSS-v4_0_1-SAQ-A-r1.pdf) and comply with the requirements listed there.

> If your payment provider categorizes you differently and requires you to fill out a different questionnaire, the guidance on this page might not apply.

Reviewer: Whether this questionnaire and the requirements apply to you depend on how your payment provider categorizes your business.

## pretix Hosted

<!-- md:hosted -->

> If you use our Software-as-a-Service offering, pretix Hosted, we are responsible for complying with many of the requirements.

Reviewer: "Software-as-a-Service" sollte bei der ersten Erwähnung aufgeschlüsselt werden und danach mit SaaS abgekürzt werden
```suggestion
If you use our SaaS offer, pretix Hosted, then the responsibility to comply with the requirements lies with us. 
```

Author: Okay, aber das "many of" kann nicht raus, denn für alle Requirements sind wir nicht verantwortlich, sonst wäre alles darunter obsolet.

Author: ```suggestion
If you use our SaaS offer, pretix Hosted, then the responsibility to comply with many of the requirements lies with us.
```

Specifically, we ensure that the technical configuration of our systems is in line with all technical requirements listed in SAQ A.
We ensure all required processes for security policies, risk management, employee training, and incident response are in place on our end.
We undergo a yearly **external certification** to ensure we are meeting all relevant requirements.

Your payment service provider might request you to submit **vulnerability scans** of your shop.

> We contract with a PCI Approved Scanning Vendor (ASV) to perform quarterly scans of our systems using sample ticket shops that are representative of our system environment for all shops.

Reviewer: ```suggestion
We have a  contract with a PCI-Approved Scanning Vendor (ASV) to perform quarterly scans of our systems using sample ticket shops that are representative of our system environment for all shops.
```

If you send your customers to the ticket shop hosted with us using a link, submitting these scan results to your payment service provider is sufficient to achieve compliance.

If you embed our [Widget](../../guides/widget.md) into your website, your payment service provider may require you to perform scans of your website as well.

> This does not apply when you use a payment provider where the credit card form always opens in a new tab or when you configure the widget to [always open in a new tab](../../guides/widget.md#always-open-a-new-tab).

Reviewer: ```suggestion
This does not apply if your payment provider's credit card form always opens in a new tab. 
You can also avoid this requirement by configuring the widget to [always open in a new tab](../../guides/widget.md#always-open-a-new-tab).
```

Should you do require a scan, there is a list of [Approved Scanning Vendors](https://www.pcisecuritystandards.org/assessors_and_solutions/approved_scanning_vendors/) on the PCI website, such as [usd AG](https://www.usd.de/en/pci-payment-security/pci-security-scans/) which is performing our scans.

Our **Responsibility Matrix** indicated which parts of the SAQ A requirements are handled by us and which are to be done by you.

> We handle most of the technical requirements, but for example requirements 12.8.1 - 12.8.5 require you to keep a list of all service providers (such as us) and regularly check that we are still PCI DSS compliant.

Reviewer: ```suggestion
We handle most of the technical requirements, but, for example, requirements 12.8.1–12.8.5 require you to keep a list of all service providers (such as us) and regularly check that we are still PCI DSS compliant.
```

You can find the latest version of all relevant documents, such as our **Attestation of Scan Compliance**, our **Attestation of Compliance** and the **Responsibility Matrix** for our relationship with you in your pretix account at :navpath:Your organizer → :fa3-wrench: Settings → PCI DSS compliance:.

> Should you require any additional documentation or have any further questions on the topic, please reach out to us at [pci@pretix.eu](mailto:support@pretix.eu).

Author: ```suggestion
Should you require any additional documentation or have any further questions on the topic, please reach out to us at [pci@pretix.eu](mailto:pci@pretix.eu).
```

## Self-hosted versions of pretix

<!-- md:community -->
<!-- md:enterprise -->

If you host pretix on your own servers, you need to ensure compliance with **all requirements** listed in the SAQ A questionnaire.

> This includes technical requirements such as using strong authentication methods, process requirements such as ensuring you always apply security updates quickly and having an incident response plan, as well as the requirement to perform **quarterly vulnerability scans** on your server through a PCI Approved Scanning Vendor (ASV).

Reviewer: ```suggestion
This includes technical requirements such as using strong authentication methods, process requirements such as applying security updates quickly and having an incident response plan, as well as the requirement to perform **quarterly vulnerability scans** on your server through an ASV. 
```
ASV wurde vorher schon aufgeschlüsselt, das ist ausreichend

Author: Aber in einem Absatz, den Self-hosted Kunden vermutlich direkt übersprungen haben, zählt das dann deiner Meinung nach trotzdem?

Reviewer: Ja, passt. Lieber einmal zu viel als einmal zu wenig :) 
Ich würde aber vorschlagen, dass eine Abkürzung pro Zwischenüberschrift höchstens einmal erklärt wird, sonst nimmt es Überhand

> The scan should at the very least include all servers used to run pretix, as well as possibly your website embeding our [Widget](../../guides/widget.md).

Reviewer: ```suggestion
The scan should include all servers used to run pretix and, if possible, your website embedding our [Widget](../../guides/widget.md).
```
Sollte der Scan die Webseite mit einschließen oder den Server, auf dem die Webseite läuft?

Author: Das ist … lustigerweise nicht ganz klar, aber soweit ich verstehe meint PCI auch spezifisch die Webseite und nicht nur den Server als ganzes.

Author: "if possible" ist nicht ganz richtig, eher ein "if applicable", geht ja nicht drum ob das scanning possible ist, sondern ob man das Widget überhaupt benutzt

```suggestion
The scan should include all servers used to run pretix and, if applicable, your website embedding our [Widget](../../guides/widget.md).
```

> The later part does not apply when you use a payment provider where the credit card form always opens in a new tab or when you configure the widget to [always open in a new tab](../../guides/widget.md#always-open-a-new-tab).

Reviewer: ```suggestion
This does not apply if your payment provider's credit card form always opens in a new tab. 
You can also avoid this requirement by configuring the widget to [always open in a new tab](../../guides/widget.md#always-open-a-new-tab).
```

There is a list of [Approved Scanning Vendors](https://www.pcisecuritystandards.org/assessors_and_solutions/approved_scanning_vendors/) on the PCI website, such as [usd AG](https://www.usd.de/en/pci-payment-security/pci-security-scans/) which is performing our scans.

> ### SAQ A eligibility criteria and custom plugins

Reviewer: Bitte einmal erklären, was das für Kriterien sind

> pretix and official pretix plugins never store any credit card numbers directly, therefore most of the SAQ A eligibility criteria are always true when using pretix with only official plugins.

Reviewer: ```suggestion
pretix and official pretix plugins never store any credit card numbers directly. 
Therefore, most of the SAQ A eligibility criteria are fulfilled if you are using pretix with official plugins only. 
```
Das Wort "most" ist an dieser Stelle wirklich gefährlich. Welche Kriterien sind nicht erfüllt?

> If you create a custom payment plugin, you need to make sure that this requirement stays true and all credit card inputs are served directly by a PCI certified payment service provider through an iframe or redirect.

Reviewer: ```suggestion
If you create a custom payment plugin, make it so that it also fulfills this requirement. 
All credit card inputs must be served directly by a PCI-certified payment provider through an iframe or redirect.
```

> Implementing payment providers where a HTML form within pretix collects the credit card information would require at least complying with the requirements of the SAQ A-EP questionnaire.

Reviewer: ```suggestion
If you want to implement a payment provider plugin with an HTML form within pretix collecting credit card information, then you need to comply with the requirements of the SAQ A-EP questionnaire. 
```

> The most recent version of the SAQ A also includes the following requirement:

Reviewer 2: Perhaps refer to a specific version or date?

Author: ```suggestion
Version 4.0.1 of the SAQ A also includes the following requirement:
```

> > The merchant has confirmed that their site is not susceptible to attacks from scripts that could affect the merchant’s e-commerce system(s).

Reviewer: ```suggestion
> The merchant has confirmed that their site is not susceptible to attacks from scripts that could affect the merchant's e-commerce system(s).
```
